### DISCUSSION TITLE
Question about fonts used in conversion from Adobe Illustrator Files

### DISCUSSION THREAD

**Discussion by Ronnieinspain** [ImageMagick/ImageMagick#6793] [Help]
### ImageMagick version

7.1.1-20

### Operating system, version and so on

Various

### Description

HI!

First of all, Kudos to all Devs who are making this awesome tool - thank you!

We are having issues with file conversion from .ai to .png, where the fonts are being modified. Looking to understand:

1. Whether fonts used in the .ai file need to be installed on the machine doing the conversion or, whether the tool automatically uses the fonts embedded in the project, if any.
2. Whether the tool can be forced to use embedded fonts, assuming there are any.

Thanks again!

**Comment by snibgo** [comment]
IM delegates the rasterization of AI files to Ghostscript. So your question is really about GS. GS has a number of options for specifying fonts at its command line.

In IM's command line, you can use "-verbose" to see what GS command it issues.

**Reply by Ronnieinspain** [reply]
Thank you @snibgo will pass this information on to the Devs.
